Formula/kyma-cli.rb

```diff
@@ -18,8 +17,12 @@ class KymaCli < Formula
   depends_on "go@1.14" => :build
 
   def install
-    system "make", "build-darwin"
-    bin.install "bin/kyma-darwin" => "kyma"
+    ldflags = %W[
+      -s -w
+      -X github.com/kyma-project/cli/cmd/kyma/version.Version=#{version}
+    ].join(" ")
+
+    system "go", "build", "-o", bin/"kyma", "-ldflags", ldflags, "-trimpath", "./cmd"
```

Comment:
Can use `*std_go_args`

Reply:
I think I have pretty override all the flags in the std_go_args (because of the bin and ldflags thing), so I am pretty ok with current build line. 

What do you think?

Reply:
ok, I think I can add `std_go_args`, just updated the PR.